force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -228,18 +287,24 @@ public with sharing class ClassesObjectsMethods {
     public class Book {
         // Declare three public instance variables - title, author, pages
         // title
+        Public String title;
         // author
+        Public String author;
         // pages
+        Public Integer pages;
     
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(String title, String author, Integer pages) {
             // Initialize the title and author
+            this.title = title;
+            this.author = author;
+            this.pages = pages;
```

Comment:
Technically, the comment instructed you to just include the title and author. This will come into play later when you are asked to initialize a book with just the author name and title (using this instructor).